File: javascript-closure-angular-client/API/Client/PaymentTransactionIn.js
goog.provide('API.Client.PaymentTransactionIn');

/**
 * @record
 */
API.Client.PaymentTransactionIn = function() {}

/**
 * @type {!API.Client.PayRecHeader}
 * @export
 */
API.Client.PaymentTransactionIn.prototype.header;

/**
 * @type {!Array<!API.Client.PaymentLinesIn>}
 * @export
 */
API.Client.PaymentTransactionIn.prototype.lines;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/PaymentTransactionIn.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// PaymentTransactionIn
    /// </summary>
    [DataContract]
    public class PaymentTransactionIn
    {
        /// <summary>
        /// Gets or Sets Header
        /// </summary>
        [DataMember(Name = "header", EmitDefaultValue = false)]
        public PayRecHeader Header { get; set; }

        /// <summary>
        /// Gets or Sets Lines
        /// </summary>
        [DataMember(Name = "lines", EmitDefaultValue = false)]
        public List<PaymentLinesIn> Lines { get; set; }
    }
}
